.github/workflows/build-installers.yml

```diff
@@ -25,15 +25,15 @@ jobs:
     - name: Compute Scalar Version
       run: |
         BRANCH=$(git branch --show-current)
-        if [ "${BRANCH:0:9" = "releases/" ]; then
-          VERSION="${BRANCH:9}".0
+        if [ "${BRANCH:0:9}" = "releases/" ]; then
+          SCALARVERSION="${BRANCH:9}".0
         else
-          VERSION=0.3.132.0
+          SCALARVERSION=0.3.132.0
         fi
-        export VERSION
```

Comment:
You cannot export `VERSION` because this (and all other environment variables) as picked up by MSBuild, and `VERSION` is screwing things up.